api/shape/toFlatValues.js

```diff
@@ -0,0 +1,25 @@
+import Shape from './Shape.js';
+import { toValue } from './toValue.js';
+
+export const toFlatValues = Shape.registerMethod(
```

Comment:
What does toFlatValues do?

Reply:
I guess the opposite of toNestedValues, but when do we want to use these?

Reply:
That's a good question -- I think those need to be cleaned up and rationalized.